src/IRMutator.cpp

```diff
@@ -392,6 +392,21 @@ Stmt IRMutator::visit(const Acquire *op) {
     }
 }
 
+Stmt IRMutator::visit(const Atomic *op) {
+    bool changed = false;
+    Stmt body = mutate(op->body);
+    if (!body.same_as(op->body)) {
```

Comment:
Don't need the changed variable here, just make `if (!body.same_as(op->body))` the condition

Reply:
Done.